### ISSUE TITLE
TypeScript - Contrast support for Scheme.

### ISSUE THREAD

**Issue by PaulHMason** [material-foundation/material-color-utilities#103]
Are there any plans to allowed a desired contrast for Scheme (light & dark) now that M3 has added contrast guidance? Another option is to just support the 3 recommended contrast buckets:

- Normal (default): 3.0
- Medium: 4.5
- High: 7.0

**Comment by guidezpl** [comment]
@pennzht I believe the plan is to deprecate Scheme, is that correct? Its direct replacement is the tonal spot scheme, which includes contrast

**Comment by pennzht** [comment]
Yes, `Scheme` will be deprecated soon. Instead of `Scheme.light(color)`, you can use:

```typescript
new SchemeTonalSpot(Hct.fromInt(color), false, 0.0)
```

You can also specify a high contrast scheme using:

```typescript
new SchemeTonalSpot(Hct.fromInt(color), false, 1.0)
```
